source/sysadmins-guide/elasticsearch-setup/index.md

```diff
@@ -112,6 +114,32 @@ return [
 ];
 ```
 
+
+### Define Elasticsaerch Dynamic Mapping
+
+By default, Shopware enables the dynamic mapping option.
+With this fields can be added dynamically to a document or to inner objects within a document, just by indexing a document containing the new field.
+
+The problem with this option is, that Shopware provides floats and integers to ElasticSearch.
```

Comment:
you should write `Elasticsearch` as this is the proper name. also in the following text